### PR TITLE
My Polls + Prettify

### PR DESCRIPTION
**Pull request by dhrvsood**
Updated PR by rebasing with master

### CODE REVIEW

File: ui/src/components/MyPollsPreview/MyPollsPreview.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    margin: 15px auto 15px;
+    padding: 5px;
+    border-radius: 10px;
+    display: block;
+    text-decoration: none;
+    text-align: left;
+}
+
+#pollName {
+    color: #FFFFFF;
```
Comment (StormFireFox1) [inline ui/src/components/MyPollsPreview/MyPollsPreview.css:12]: I'm not sure this `color` rule is necessary; make sure to try the code without the rule and see if it still has the same color. Black is the default color for text, and that's what is interesting here.

File: ui/src/components/MyPollsPreview/MyPollsPreview.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { render } from 'react-dom';
+import './MyPollsPreview.css';
+
+class MyPollsPreview extends React.Component {
+    render() {
+        const isFinished = (this.props.pollId != -1);
```
Comment (StormFireFox1) [inline ui/src/components/MyPollsPreview/MyPollsPreview.jsx:7]: You should add a TODO here so we can find it easier when we'll write the middleware:
```suggestion
        // TODO add check for "active" in particular poll's properties
        const isFinished = (this.props.pollId != -1);
```

File: ui/src/components/MyPollsPreview/MyPollsPreview.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { render } from 'react-dom';
+import './MyPollsPreview.css';
+
+class MyPollsPreview extends React.Component {
```
Comment (StormFireFox1) [inline ui/src/components/MyPollsPreview/MyPollsPreview.jsx:5]: How come this specific component is a class component? Every other component here is functional, as well as most in the codebase. We'd best rewrite this to a functional component. Considering its simplicity, there aren't that many changes.

File: ui/src/components/MyPollsPreview/MyPollsPreview.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    render() {
+        const isFinished = (this.props.pollId != -1);
+
+        // default unfinished draft
+        var backgroundColor = '#62B0FF';
```
Comment (StormFireFox1) [inline ui/src/components/MyPollsPreview/MyPollsPreview.jsx:10]: We can modify this when we'll attach middleware, but essentially, you'll want to define CSS styles for draft and active polls and selectively assign styles below by checking the "active" property. I'll explain how when we get to the middleware writing, which will be very soon.

**Review by StormFireFox1 — CHANGES_REQUESTED** [review]
Page looks nice!

Few changes, though. Suggestions are added where possible, as usual.

File: ui/src/components/MyPollsPreview/MyPollsPreview.css (hunk 1 above)
Comment (dhrvsood) [reply ui/src/components/MyPollsPreview/MyPollsPreview.css:12]: The preview components have to be wrapped in <a> tags in order to be clickable, which defaults the colour to blue. The #FFFFFF rule is necessary to make it look like the Figma.

**Review by StormFireFox1 — APPROVED** [review]
Looks good! "Squash and Merge" it!

**Pull request merged**